src/hashfunctions.c

```diff
@@ -261,6 +295,17 @@ static Int InitKernel(void)
 static Int InitLibrary(void)
 {
     InitGVarFuncsFromTable(GVarFuncs);
+
+    // We  Reuse the 1 argument function for 1 argument, and also for
+    // handling a list of arguments
+    Obj gvar = NewFunctionC("DATA_HASH_FUNC_RECURSIVE", -1L, "arg",
```

Comment:
Just blindly copied it from some other call of `NewFunctionC`, will have a look if there is any reason this is actually required.